epoxy-compose/src/main/java/com/airbnb/epoxy/ComposeInterop.kt

```diff
@@ -11,27 +11,60 @@ import androidx.compose.ui.viewinterop.AndroidView
 class ComposeEpoxyModel(
     private val composeFunction: @Composable () -> Unit
 ) : EpoxyModelWithView<ComposeView>() {
+    private lateinit var keys: Array<Any>
+
     override fun buildView(parent: ViewGroup): ComposeView = ComposeView(parent.context)
 
     override fun bind(view: ComposeView) {
         super.bind(view)
         view.setContent(composeFunction)
     }
+
+    override fun equals(other: Any?): Boolean {
+        if (other === this) return true
+        if (other !is ComposeEpoxyModel) return false
+        if (this.hashCode() != other.hashCode()) return false
+
+        for (i in keys.indices) {
+            if (keys[i] != other.keys[i]) return false
+        }
+
+        return true
+    }
+
+    override fun hashCode(): Int {
+        var code = super.hashCode()
+
+        keys.forEach {
+            code = 31 * code + it.hashCode()
+        }
+
+        return code
+    }
+
+    /**
+     * This key will contribute to calculate hashcode & equals
+     * */
+    fun keys(vararg keys: Any) {
```

Comment:
seems like this should go in the constructor to enforce it is set. (it is lateinit right now which will crash if not set)